ProjectVoyage/Cell/ChatCell.m

```diff
@@ -8,27 +8,21 @@
 #import "ChatCell.h"
 #import "Util.h"
 
-@implementation ChatCell
-
-- (void)awakeFromNib {
-    [super awakeFromNib];
-    // Initialization code
-}
-
-
-- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
-    [super setSelected:selected animated:animated];
-    // Configure the view for the selected state
-}
+@interface ChatCell()
+@property (nonatomic, strong) IBOutlet UILabel *recipientLabel;
+@property (nonatomic, strong) IBOutlet UILabel *descriptionLabel;
+@property (nonatomic, strong) IBOutlet UIImageView *recipientImage;
+@property (nonatomic, strong) IBOutlet UILabel *dateChat;
+@end
 
+@implementation ChatCell
 
 - (void) setChat:(Chat *)chat {
     _chat = chat;
-    self.recipientImage.image = [UIImage imageWithData:[chat.recipientImage getData]];
-    self.recipientLabel.text = chat.recipientName;
-    self.descriptionLabel.text = chat.chatDescription;
-    
-    self.dateChat.text = [Util formatDateString:chat.createdAt];
```

Comment:
just a nit

in objective c, you can use the underscore infront of ivars to reference them